sysl2/sysl/sysl_test.go

```diff
@@ -211,32 +203,47 @@ func TestMain2JSONStdout(t *testing.T) {
 }
 
 func TestMain2BadMode(t *testing.T) {
-	var stdout, stderr bytes.Buffer
-	rc := main2(&stdout, &stderr, []string{"sysl", "-o", "-", "-mode", "BAD", "tests/args.sysl"}, main3)
+	rc := main2([]string{"sysl", "-o", "-", "-mode", "BAD", "tests/args.sysl"}, main3)
 	assert.NotZero(t, rc)
 
 	_, err := os.Stat("-")
 	assert.True(t, os.IsNotExist(err), "Should not have created file '-'")
 }
 
 func TestMain2BadLog(t *testing.T) {
-	var stdout, stderr bytes.Buffer
-	rc := main2(&stdout, &stderr, []string{"sysl", "-o", "-", "-log", "BAD", "tests/args.sysl"}, main3)
+	rc := main2([]string{"sysl", "-o", "-", "-log", "BAD", "tests/args.sysl"}, main3)
 	assert.NotZero(t, rc)
 
 	_, err := os.Stat("-")
 	assert.True(t, os.IsNotExist(err), "Should not have created file '-'")
 }
 
+func TestMain2WithBlackboxParams(t *testing.T) {
+	testHook := test.NewGlobal()
+	main2([]string{"sd", "-s", "MobileApp <- Login", "-o", "tests/call.png", "-b", "Server <- DB=call to database",
+		"-b", "Server <- Login=call to database", "tests/call.sysl"}, main3)
+	assert.Equal(t, logrus.WarnLevel, testHook.LastEntry().Level)
+	assert.Equal(t, "blackbox 'Server <- DB' not hit in app ' Global :: '\n", testHook.LastEntry().Message)
+	testHook.Reset()
+}
+
+func TestMain2WithBlackboxSysl(t *testing.T) {
+	testHook := test.NewGlobal()
+	main2([]string{"sd", "-o", "%(epname).png", "tests/blackbox.sysl", "-a", "Project :: Integrations"}, main3)
```

Comment:
there should be an error in this case. The endpoint in `Project :: Integrations` does not have any call statements.

The correct project to pass is  `Project :: Sequences`.